vispy/geometry/curves.py

```diff
@@ -0,0 +1,343 @@
+# ----------------------------------------------------------------------------
+#  Anti-Grain Geometry (AGG) - Version 2.5
+#  A high quality rendering engine for C++
+#  Copyright (C) 2002-2006 Maxim Shemanarev
+#  Contact: mcseem@antigrain.com
+#           mcseemagg@yahoo.com
+#           http://antigrain.com
+#
+#  AGG is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
```

Comment:
Agg 2.4 is MIT while Agg 2.5 is GPL. I think I translated the Agg 2.4 and also, since it is a complete rewrite (from me), I think it is ok to put a BSD license.

See also: http://sourceforge.net/p/vector-agg/mailman/message/30696434/